### DISCUSSION TITLE
How to check if Zigbee device acts correctly as bridge?

### DISCUSSION THREAD

**Discussion by jbrepogmailcom** [Koenkk/zigbee2mqtt#7234] [Devices]
Hello, is it possible to see from the log if Zigbee device correctly acts as bridge, i.e. correctly forwards packets to other devices? I have Zigbee socket, always powered and I am courious to see if it routes packets to other devices as it should.

Thanks, Jan

**Comment by FlexxFR** [comment]
Yes, you could restart the zigbee2mqtt add-on and you see it in the log files.
Or use the webUI and select the device, All data is there, including the router mode
